.github/workflows/nightly-e2e.yaml

```
@@ -1418,6 +1419,42 @@ jobs:
+  # Regression test for #2342: verifies health probes work correctly when
+  # device auth is enabled (the default). Previously `curl -sf` treated
+  # HTTP 401 as failure, causing false "Health Offline" readings.
+  # Validates: /health returns 200, / returns 401, status != Offline,
+  # gateway recovery with device auth, port forward liveness.
+  device-auth-health-e2e:
+    if: >-
+      github.repository == 'NVIDIA/NemoClaw' &&
+      (github.event_name != 'workflow_dispatch' ||
+       inputs.jobs == '' ||
+       contains(format(',{0},', inputs.jobs), ',device-auth-health-e2e,'))
+    runs-on: linux-amd64-cpu4
```

actionlint (1.7.12)

[error] label "linux-amd64-cpu4" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)